deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/resource/Hotfixs.java

```diff
@@ -84,7 +84,11 @@ public HotfixBean get(@PathParam("id") String id) throws Exception {
     @Path("/{id : [a-zA-Z0-9\\-_]+}")
     public void update(@Context SecurityContext sc, @PathParam("id") String id,
         HotfixBean hotfixBean) throws Exception {
-        authorizer.authorize(sc, new Resource(Resource.ALL, Resource.Type.SYSTEM), Role.OPERATOR);
+        try {
+            authorizer.authorize(sc, new Resource(Resource.ALL, Resource.Type.SYSTEM), Role.PUBLISHER);
+        } catch (Exception e) {
+            LOG.info("Failed to authorize hotfix {} with {}.", id, hotfixBean);
```

Comment:
We should let the exception throw out instead of catch and swallow.

Reply:
Good point. Changed to throw out exception.